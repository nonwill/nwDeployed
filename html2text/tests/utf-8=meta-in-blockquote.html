<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  </head>
  <body>
    <div class="moz-cite-prefix">On 4/5/20 5:54 AM, J N
      wrote:<br>
    </div>
    <blockquote type="cite"
      cite="mid:0c944ee8-8a78-fe56-8311-5f30f70285f3@gentoo.org">
      <meta http-equiv="content-type" content="text/html; charset=UTF-8">
      <div class="add"><tt># Not maintained in Gentoo, doesn't build for
          2 years, has only</tt></div>
      <div class="add"><tt># deprecated version present in Gentoo. Has a
          huge number of open</tt></div>
      <div class="add"><tt># bugs. Removal in 30 days. #642952</tt></div>
      <div class="add"><tt>www-misc/zoneminder</tt></div>
    </blockquote>
    <font size="+1">This appears to be an active project.</font><font
      size="+1">  There is a plugin for <b>zoneminder</b> and <b>zmserver</b>
      in <b>mythtvplugin</b><b>s</b>.  When I finish the <b>mythtv</b>
      version 31.0 bump I will start maintaining this package.</font><br>
  </body>
</html>
