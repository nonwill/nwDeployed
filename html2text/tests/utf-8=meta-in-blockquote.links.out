utf-8=meta-in-blockquote.default.out